src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -509,6 +516,10 @@ public override void Init(ICellEditorData owningEditor, bool fresh)
             behaviourTabButton.Visible = false;
             behaviourEditor.Visible = false;
         }
+
+        // after the if multicellular check so it resets if this is a single-cell editor
+        // but the previously opened editor was multicell (ie save and switch to new file)
+        UpdateTooltipMPCostFactors();
```

Comment:
So this is also required to run for the multicellular editor? If not, I'd place this in an else. If this is required to run anyway, I'd change the above comment to be something like "Make sure we have up to date cost factors set on tooltips" to emphasize that fact.

Reply:
The reason I did it this way (and also the first change you requested with the extra variable mpDisplay) was because otherwise the reduced costs were persistent on exiting to menu and opening a new or saved game.

I don't think tooltip objects are garbage collected when exiting the editor and re-created, they're still in the tooltip manager when the main gameplay is running. I didn't want to make major changes to the whole tooltip system in this PR so this was my solution - as well as the separate mpDisplay variable because I wanted the original cost to only ever be updated from the JSON so opening the wrong sequence of editors couldn't lead to a state where the MP cost in tooltips was incorrect (more of a future proofing than anything).

Let me know what your thoughts are!

Reply:
Yes, the tooltips are in a separate manager object, I'm not criticising the need to update the tooltips always here. I'm suggesting a simpler explanation comment. Mainly because to me this current wording is misleading regarding how the editor object is handled. The editor object tree is in fact deleted each time the editor is exited so that can't have invalid state, but as I said the tooltip manager owns the tooltips which persist throughout the game.